apps/hosts/views.py

```diff
@@ -13,10 +18,32 @@
                      ChallengeHostTeam,)
 from .serializers import (ChallengeHostSerializer,
                           ChallengeHostTeamSerializer,
-                          InviteHostToTeamSerializer,
                           HostTeamDetailSerializer,)
 
 
+SECRET_KEY = "hotchocolates"
+
+
+def ENCODE_DATA(data):
```

Comment:
Please change the method name to `encode_data()` and you should move this method to `utils.py` in the same directory.